src/Chapter1.hs

```diff
@@ -568,7 +575,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+	| c == 'a' || c == 'A' = True
+	| c == 'e' || c == 'E' = True
+	| c == 'i' || c == 'I' = True
+	| c == 'o' || c == 'O' = True
+	| c == 'u' || c == 'U' = True
+	| otherwise = False
```

Comment:
That's a great implementation! 👍🏻 In the future, you can improve it by using standard function [toLower](http://hackage.haskell.org/package/base-4.14.1.0/docs/GHC-Unicode.html#v:toLower), but we explain imports of other functions only in the next chapter, so no need to do anything now except being aware of alternatives 🙂